.github/workflows/lint.yaml

```diff
@@ -0,0 +1,36 @@
+# validate any terraform file changes
+name: Terraform lint
+
+env:
+  TERRAFORM_VERSION: "1.2.5"
+  TERRAGRUNT_VERSION: "0.38.6"
+
+on:
```

Comment:
Suggestion: could we also add [`workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)? I usually find it very useful when testing something or re-running on flakes.

Reply:
Thanks for your suggestions, I have updated it.